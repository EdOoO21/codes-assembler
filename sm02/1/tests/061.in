-628 140 -557 658 857 -344 -400 -343 646 419 -944 -5 -268 -265 -867 -170 -56 -325 995 -657 -315 293 -988 830 29 410 -813 300 -644 -309 -946 -113 651 -705 -739 492 -175 -393 854 -403 511 728 -303 -98 -348 265 -238 558 678 286 280 -635 601 460 259 -471 -141 -166 -521 -733 -630 -881 108
