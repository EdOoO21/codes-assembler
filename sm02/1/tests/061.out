108
-881
-630
-733
-521
-166
-141
-471
259
460
601
-635
280
286
678
558
-238
265
-348
-98
-303
728
511
-403
854
-393
-175
492
-739
-705
651
-113
-946
-309
-644
300
-813
410
29
830
-988
293
-315
-657
995
-325
-56
-170
-867
-265
-268
-5
-944
419
646
-343
-400
-344
857
658
-557
140
-628
